requirements/admin.txt.in

```diff
@@ -0,0 +1,53 @@
+# The following list of immediate dependencies can be produced by install
+# snakefood and then running the following command line.
+#
+# Get a list of standard library packages so that we can omit those from our
+# requirements list.
+#
+# pip install stdlib-list
+#
+# python -c <<EOF  | cut --delimiter '.' --fields 1 | sort | uniq > std_packages
+#     from stdlib_list import stdlib_list
+#     print "\n".join(stdlib_list("2.7"))
+# EOF
+#
+# Install snakefood and use it to extract the non-dynamic Flocker imports.
+#
+# pip install snakefood
+#
+# sfood-imports --unified \
+#               --ignore acceptance \
+#               --ignore test \
+#               --ignore functional \
+#               admin \
+# | cut --delimiter '.' \
+#       --fields 1 \
+# | sort \
+# | uniq \
+# | egrep --file std_packages \
+#         --line-regexp \
+#         --invert-match
+#
+# Then inspect the list by hand. Remove any false positives and replace package
```

Comment:
What are false positives? Why do we sometimes have to do the package / pyPI package transformation?